modules/base.nix

```diff
@@ -120,9 +120,9 @@ with lib;
           server_names_hash_max_size 1024;
           server_names_hash_bucket_size  64;
 
-          access_log stderr;
-          error_log stderr;
```

Comment:
Did you have to change this? The error log should not go to a FS directory, as we have had full disks in the past due to this.